test/e2e/far_e2e_test.go

```diff
@@ -40,11 +40,11 @@ const (
 	forced client.GracePeriodSeconds = 0
 	//TODO: try to minimize timeout
 	// eventually parameters
-	timeoutLogs     = 3 * time.Minute
-	timeoutReboot   = 6 * time.Minute  // fencing with fence_aws should be completed within 6 minutes
-	timeoutDeletion = 10 * time.Second // this timeout is used after all the other steps have been succesfult
-	pollDeletion    = 250 * time.Millisecond
-	pollInterval    = 10 * time.Second
+	timeoutLogs   = "3m0s"
+	timeoutReboot = "6m0s" // fencing with fence_aws should be completed within 6 minutes
+	timeoutShort  = "5s"   // this timeout is used after all the other steps have been succesfult
+	pollShort     = "250ms"
```

Comment:
why change the var names with suffix `Short` , I think in this context long/short time are very subjective (especially since the "long" time doesn't exist anymore as a baseline to compare to) terms so I prefer a more meaningful name.

Reply:
I agree, it was just short from my POV.